src/js/utils/filter-source.js

```diff
@@ -55,13 +56,15 @@ const filterSource = function(src) {
  * @return {Tech~SourceObject}
  *        src Object with known type
  */
-function checkMimetype(src) {
+function fixSource(src) {
   const mimetype = getMimeType(src.src);
 
   if (!src.type && mimetype) {
     src.type = mimetype;
   }
 
+  src.src = getAbsoluteURL(src.src);
```

Comment:
why change the source to be an absolute url?

Reply:
I don't think we should, I think this was leftover from some testing I was doing with absolute vs relative sources.